cookiecutter/{{cookiecutter.github_repo_name}}/pyproject.toml

```diff
@@ -0,0 +1,146 @@
+# License: MIT
+# Copyright © {% now 'utc', '%Y' %} Frequenz Energy-as-a-Service GmbH
+
+[build-system]
+requires = [
+  "setuptools == 67.7.2",
+  "setuptools_scm[toml] == 7.1.0",
+  "frequenz-repo-config[{{cookiecutter.type}}] == 0.2.0",
```

Comment:
This version is unreleased for now. It is not ideal to have a dependency on itself, but I don't see this as a big problem as the template will be probably used from the `main` branch and not a tag anyway. If it becomes a problem in the future we might think about splitting out the cookiecutter to a separate repo, but I like the fact that everything that needs to be updated if the way to configure a repo changes lives in one central place is not scattered on multiple repos.